### ISSUE TITLE
Support CMI

### ISSUE THREAD

**Issue by Adri417** [LMBishop/Quests#327]
### Reason for Addition

Support cmi for Money earn and money total similar to essentials because i use cmi and i don't have that solution :/

### Description of Feature

Support cmi for Money earn and money total similar to essentials because i use cmi and i don't have that solution :/

### Alternative Solutions

I don't find alternatinve solution :/

### Agreements

- [X] I am running the latest version of Quests
- [X] I am certain this issue is unique and a similar issue is not currently open

**Comment by NicoNekoDev** [comment]
You could use the command `money give <player> <amount>` instead of `eco give`

**Comment by Adri417** [comment]
Yes but i want a condition for win a quest exemple : reach 5000$

**Comment by NicoNekoDev** [comment]
Unsure why the author didn't add support directly for Vault.

**Comment by LMBishop** [comment]
Vault is very old has no events API, for some stupid reason, so I just integrated with the most popular money plugin which does

**Comment by NicoNekoDev** [comment]
Why not use Treasury then?
https://www.spigotmc.org/resources/treasury.99531/

**Comment by LMBishop** [comment]
Doesn't seem like either Essentials or CMI support it (yet).

**Comment by PovilasLT** [comment]
You can use %cmi_user_balance% placeholder with [placeholderapi_evaluate](https://github.com/LMBishop/Quests/wiki/Task-Configuration-Layout#placeholderapi_evaluate).

**Comment by Adri417** [comment]
yes but the condition for win the quest cmi money don't work
I need a condition that when we reach a certain amount we pass the quest with cmi
